### DISCUSSION TITLE
Anyone affected by Zalastax no-one-left-behind?

### DISCUSSION THREAD

**Discussion by jsiegenthaler** [npm/feedback#858] [Registry] [answered]
The package https://www.npmjs.com/package/no-one-left-behind published by Zalastax appears to have made itself dependent to every single npm package, for no reason that I can understand.

This has affected me and blocked me from any unpublish actions on my packages.

Anyone else affected?

**Comment by ayan4m1** [comment]
Tried reporting this to NPM using their contact form, it redirected me to a 404 page twice. 🤦 

EDIT: There's already a second individual doing this under the name "infinitebrahminuniverse"

**Reply by jsiegenthaler** [reply]
Try via https://www.npmjs.com/support

**Reply by ayan4m1** [reply]
Yes, that's the form I used, it failed twice yesterday but worked this morning.

**Comment by MylesBorins** [comment, marked as answer]
The package has been removed

**Reply by jsiegenthaler** [reply]
Thanks very much @MylesBorins, I really appreciate it.
 
Note that whilst the top level package has been removed, the underlying child packages (>1000 of them) that spawned from the top level package still exist. 
Example:
https://www.npmjs.com/package/@zalastax/nolb-a
...
https://www.npmjs.com/package/@zalastax/nolb-h
https://www.npmjs.com/package/@zalastax/nolb-ho
https://www.npmjs.com/package/@zalastax/nolb-hom
https://www.npmjs.com/package/@zalastax/nolb-home
https://www.npmjs.com/package/@zalastax/nolb-homeb
https://www.npmjs.com/package/@zalastax/nolb-homebr
https://www.npmjs.com/package/@zalastax/nolb-homebri
https://www.npmjs.com/package/@zalastax/nolb-homebrid
https://www.npmjs.com/package/@zalastax/nolb-homebridg
https://www.npmjs.com/package/@zalastax/nolb-homebridge
https://www.npmjs.com/package/@zalastax/nolb-homebridge-
https://www.npmjs.com/package/@zalastax/nolb-homebridge-a
https://www.npmjs.com/package/@zalastax/nolb-homebridge-c
https://www.npmjs.com/package/@zalastax/nolb-homebridge-d
https://www.npmjs.com/package/@zalastax/nolb-homebridge-e --> this is the package blocking me
https://www.npmjs.com/package/@zalastax/nolb-homebridge-f

and so on.

In addition to the above, another user has run the same script creating the same tree of locking dependents.
you will find the same tree above under the following user @infinitebrahmanuniverse, example:
...
https://www.npmjs.com/package/@infinitebrahmanuniverse/nolb-homebridge-c
https://www.npmjs.com/package/@infinitebrahmanuniverse/nolb-homebridge-d
https://www.npmjs.com/package/@infinitebrahmanuniverse/nolb-homebridge-e
...
You can crawl backwards and forwards and discover the same structure of thousands of package dependencies.

Thanks in advance for cleaning this mess.

**Reply by MylesBorins** [reply]
I'm surfacing this to our Trust and Safety team, thank you for helping with the investigation
